charts/ai-software-templates/chatbot/0.1.0/Chart.yaml

```diff
@@ -1,4 +1,7 @@
 apiVersion: v2
+kubeVersion: ">= 1.27.0-0"
+annotations:
+  charts.openshift.io/name: Chatbot AI Software Template
```

Comment:
Nit: This is probably too late to change for this iteration of the Helm chart, but we might want to avoid using the word "template" throughout the helm chart. 

IMO, if I'm someone just looking at this chart (and not knowing it was adapted from an RHDH software template), I'd be wondering it's called a software template here.

Reply:
Yeah +1. We mention it on the readme that is an attempt to convert the RHDH AI Software Templates but given the future direction I think we can remove this.